samples/drivers/adc/adc_sequence/boards/frdm_mcxc242.overlay

```diff
@@ -4,7 +4,7 @@
  * Copyright 2024 NXP
  */
 
- / {
+/ {
```

Comment:
Fixups like this should be done in the original commit introducing the file, not in a later commit in the PR.

Reply:
Sure. I did it by mistake. Thank you for noticing it.